Buildings/Applications/DataCenters/ChillerCooled/Examples/IntegratedPrimarySecondaryEconomizer.mo

```diff
@@ -2,18 +2,34 @@ within Buildings.Applications.DataCenters.ChillerCooled.Examples;
 model IntegratedPrimarySecondaryEconomizer
   "Example that demonstrates a chiller plant with integrated primary-secondary side economizer"
   extends Modelica.Icons.Example;
-
-  extends
-    Buildings.Applications.DataCenters.ChillerCooled.Examples.BaseClasses.PartialDataCenter(
+  extends Buildings.Applications.DataCenters.ChillerCooled.Examples.BaseClasses.PostProcess(
+    freCooSig(
+      y=if cooModCon.y == integer(Buildings.Applications.DataCenters.Types.CoolingModes.FreeCooling)
+      then 1 else 0),
+    parMecCooSig(
+      y=if cooModCon.y == integer(Buildings.Applications.DataCenters.Types.CoolingModes.PartialMechanical)
+      then 1 else 0),
+    fulMecCooSig(
+      y=if cooModCon.y == integer(Buildings.Applications.DataCenters.Types.CoolingModes.FullMechanical)
+      then 1 else 0),
+    PHVAC(y=cooTow[1].PFan + cooTow[2].PFan + pumCW[1].P + pumCW[2].P + sum(
+          chiWSE.powChi + chiWSE.powPum) + sum(secPum.P) + ahu.PFan + ahu.PHea),
+    PIT(y=roo.QSou.Q_flow));
+  extends Buildings.Applications.DataCenters.ChillerCooled.Examples.BaseClasses.PartialDataCenter(
     redeclare Buildings.Applications.DataCenters.ChillerCooled.Equipment.IntegratedPrimarySecondary chiWSE(
         addPowerToMedium=false,
         perPum=perPumPri,
         energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
-        use_controller=false),
-    pumCW(each energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
-        each use_inputFilter=true),
-    ahu(energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
-        use_inputFilterValve=false));
+        use_controller=false,
+      riseTimePump=30),
+    pumCW(each energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial, each
+        use_inputFilter=false),
```

Comment:
Are these resetting of the parameters needed? Better set it to the right value in the base class.